Tasks/PackerBuildV1/task.json

```diff
@@ -0,0 +1,325 @@
+{
+    "id": "845fd4f4-642d-4694-8514-047948a5a556",
+    "name": "PackerBuild",
+    "friendlyName": "Build Machine Image",
+    "description": "Build machine image using Packer. This image can be used for Azure Virtual machine scale set deployment",
+    "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?linkid=845329)",
+    "category": "Deploy",
+    "releaseNotes": "- Works with cross-platform agents (Linux, macOS, or Windows)\n- Supports building images for both Windows and Linux. Supports both managed and unmanaged VM images. These images can be used to deploy Azure Virtual machine scale set.",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "demands": [],
+    "preview": "true",
+    "minimumAgentVersion": "2.0.0",
+    "groups": [
+        {
+            "name": "AzureDetails",
+            "displayName": "Azure Details",
+            "isExpanded": true,
+            "visibleRule": "templateType = builtin"
+        },
+        {
+            "name": "DeploymentInputs",
+            "displayName": "Deployment Inputs",
+            "isExpanded": true,
+            "visibleRule": "templateType = builtin"
+        },
+        {
+            "name": "Advanced",
+            "displayName": "Advanced",
+            "isExpanded": false,
+            "visibleRule": "templateType = builtin"
+        },
+        {
+            "name": "Output",
+            "displayName": "Output",
+            "isExpanded": true
+        }
+    ],
+    "inputs": [
+        {
+            "name": "templateType",
+            "type": "pickList",
+            "label": "Packer template",
+            "required": true,
+            "helpMarkDown": "Select whether you want the task to auto generate Packer template or use custom template provided by you.",
+            "options": {
+                "builtin": "Auto generated",
+                "custom": "User provided"
+            },
+            "defaultValue": "builtin"
+        },
+        {
+            "name": "isManagedImage",
```

Comment:
@azooinmyluggage just check the location of this input in task. It seems okay to me.

Reply:
Looks good to me too.